packages/svelte/types/index.d.ts

```diff
@@ -238,7 +238,8 @@ declare module 'svelte' {
 	 * Convenience type to get the properties that given component exports.
 	 *
 	 * Example: Typing the `bind:this` for a component named `MyComponent`
-	 * ```
+	 *
+	 * ```ts
```

Comment:
Should this be `svelte` instead of `ts`? I don't know if that works in VSCode or other editors.

Reply:
```suggestion
	 * ```svelte
```